src/logic/community/community.ts

```diff
@@ -39,6 +45,19 @@ export function createCommunityComponent(
       }
     },
 
+    getPublicCommunities: async (options: GetCommunitiesOptions): Promise<GetCommunitiesResult<PublicCommunity>> => {
```

Comment:
What do you think about, maybe another one, but seems we will have the `public | private` concept it may confuses future devs, not strong opinion btw
```suggestion
    getCommunitiesWithPublicInformation: async (options: GetCommunitiesOptions): Promise<GetCommunitiesResult<PublicCommunity>> => {
```